library/datetime.po

```diff
@@ -809,12 +811,17 @@ msgid ""
 "emitted by :meth:`date.isoformat`. Specifically, this function supports "
 "strings in the format(s) ``YYYY-MM-DD``."
 msgstr ""
+"Renvoie une :class:`date` correspondant à *date_string* dans le format émis "
+"par :meth:`date.isoformat`. Spécifiquement, cette fonction supporte des "
```

Comment:
Dans la documentation, on traduit généralement "to support" par "gérer".

Reply:
:+1: